<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiotrackDockWidgetBase</class>
 <widget class="QDockWidget" name="RadiotrackDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radiotrack</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="init_tab">
       <attribute name="title">
        <string>Principal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="importButton">
             <property name="toolTip">
              <string>Import data from a compatible CSV file.
Any previously loaded data will be replaced.</string>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveAsButton">
             <property name="toolTip">
              <string>Export the table above as a CSV file.</string>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="toolTip">
              <string>Delete the table above and the plugin's map layers.</string>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>45</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Project:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="currentProjectText">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Path of the last imported CSV file.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filters">
       <attribute name="title">
        <string>Filters</string>
       </attribute>
       <widget class="QCheckBox" name="filter1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>92</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Filter1</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="StartingDate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>110</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="EndingDate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>110</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeStart">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>194</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="documentation">
       <attribute name="title">
        <string>Documentation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="documentationText"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
